### ISSUE TITLE
Calling "events.all" on a calendar results in Inbox::ResourceNotFound

### ISSUE THREAD

**Issue by singjsong** [nylas/nylas-ruby#23]
As the title says. Tried this on different calendars across different accounts. Stacktrace:
```
Inbox::ResourceNotFound: Inbox::ResourceNotFound
	from /usr/local/rvm/gems/ruby-2.1.2/gems/inbox-0.15.1/lib/inbox.rb:29:in `interpret_http_status'
	from /usr/local/rvm/gems/ruby-2.1.2/gems/inbox-0.15.1/lib/inbox.rb:35:in `interpret_response'
	from /usr/local/rvm/gems/ruby-2.1.2/gems/inbox-0.15.1/lib/restful_model_collection.rb:133:in `block in get_model_collection'
	from /usr/local/rvm/gems/ruby-2.1.2/gems/rest-client-1.7.2/lib/restclient/request.rb:493:in `call'
	from /usr/local/rvm/gems/ruby-2.1.2/gems/rest-client-1.7.2/lib/restclient/request.rb:493:in `process_result'
	from /usr/local/rvm/gems/ruby-2.1.2/gems/rest-client-1.7.2/lib/restclient/request.rb:421:in `block in transmit'
	from /usr/local/rvm/rubies/ruby-2.1.2/lib/ruby/2.1.0/net/http.rb:853:in `start'
	from /usr/local/rvm/gems/ruby-2.1.2/gems/rest-client-1.7.2/lib/restclient/request.rb:413:in `transmit'
	from /usr/local/rvm/gems/ruby-2.1.2/gems/rest-client-1.7.2/lib/restclient/request.rb:176:in `execute'
	from /usr/local/rvm/gems/ruby-2.1.2/gems/rest-client-1.7.2/lib/restclient/request.rb:41:in `execute'
	from /usr/local/rvm/gems/ruby-2.1.2/gems/rest-client-1.7.2/lib/restclient.rb:65:in `get'
	from /usr/local/rvm/gems/ruby-2.1.2/gems/inbox-0.15.1/lib/restful_model_collection.rb:132:in `get_model_collection'
	from /usr/local/rvm/gems/ruby-2.1.2/gems/inbox-0.15.1/lib/restful_model_collection.rb:58:in `range'
	from /usr/local/rvm/gems/ruby-2.1.2/gems/inbox-0.15.1/lib/restful_model_collection.rb:42:in `all'
```

Personally not an issue for me since we don't utilize that method chain, but still a bug nonetheless.

**Comment by khamidou** [comment]
Thanks for the report, Jake. I'll try to fix this next week.

**Comment by khamidou** [comment]
Fixed in 6555a7a.
